javascript/javascript3/week2/Movies/movies_array.js

```diff
@@ -0,0 +1,39 @@
+//1. Create an array of bad movies
+const apiUrl = `https://tinyurl.com/y9zck5xx`;
```

Comment:
Always try to name things appropriately. apiUrl is just a generic name that could mean anything